tests/sqllogictests/suites/tpcds/queries.test

```diff
@@ -3432,7 +3432,7 @@ NULL NULL NULL NULL
 
 
 # Q14
-onlyif mysql
+onlyif todo
```

Comment:
Why disable Q14?

Reply:
The result of Q14 in cluster mode is wrong.